src/Repository/AdherentRepository.php

```diff
@@ -525,4 +527,40 @@ public function countByGenderManagedBy(Adherent $referent): array
             ->getArrayResult()
         ;
     }
+
+    public function countSupervisorsByGenderForReferent(Adherent $referent): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        $result = $this->createQueryBuilder('adherent', 'adherent.gender')
+            ->select('adherent.gender, COUNT(DISTINCT adherent) AS count')
+            ->join('adherent.memberships', 'membership')
+            ->join(Committee::class, 'committee', Join::WITH, 'committee.uuid = membership.committeeUuid')
+            ->join('committee.referentTags', 'tag')
+            ->where('tag.id IN (:tags)')
```

Comment:
can you try to use `andWhere('committee.referentTags IN (...))` directly and remove the join `->join('committee.referentTags', 'tag')`?

Reply:
It does not work and throws en error: `Error: Invalid PathExpression. StateFieldPathExpression or SingleValuedAssociationField expected.`